src/account/SignIn.jsx

```diff
@@ -1,20 +1,16 @@
-
 import React from 'react';
-import { getAuth, signInWithEmailAndPassword } from 'firebase/auth';
-
+import { getAuth, signInWithEmailAndPassword} from 'firebase/auth';
 
 const auth = getAuth();
 
 export default function SignInScreen(){
```

Comment:
If it is a component, you can name it `SignIn` instead of `SignInScreen`.

Reply:
changed it